apps/mull-ui/src/app/pages/messages/event-chat/event-chat.tsx

```diff
@@ -130,20 +151,75 @@ export const EventChat = ({ history, channelName, restrictChatInput }: EventChat
       }
     },
   });
-  if (error) {
-    return <Redirect to={ROUTES.LOGIN} />;
-  }
+
+  const updateContainerStyle = () => {
+    const chatInputContainer = document.getElementsByClassName(
+      'chat-input-container'
+    )[0] as HTMLDivElement;
+    if (chatInputContainer) {
+      const margin =
+        document.documentElement.clientHeight - chatInputContainer.getBoundingClientRect().top;
+      setContainerStyle({ marginBottom: margin + 'px' });
+      setTimeout(() => {
+        window.scrollTo({ top: eventChatRef.current.clientHeight + 1000 });
+      }, 100);
+    }
+  };
+
+  const inputOnFocus = () => {
+    if (window.innerWidth < 800) {
+      const botNav = document.getElementsByClassName('bot-nav-container')[0] as HTMLDivElement;
+      botNav.style.display = 'none';
+
+      const chatInputContainer = document.getElementsByClassName(
+        'chat-input-container'
+      )[0] as HTMLDivElement;
+
+      chatInputBot.current = chatInputContainer.style.bottom;
+
+      chatInputContainer.style.bottom = '0';
+
+      updateContainerStyle();
+    }
+  };
+  const inputOnBlur = () => {
```

Comment:
Had some issue with the bottom nav bar not coming back if I don't click out of the event chat input

https://user-images.githubusercontent.com/33991121/114085066-dca47a00-987e-11eb-93a0-ce150b630bae.mov

Reply:
Should be fixable by force blurring the chat input once you click enter. I'll see what I can do